leaflet-treering.js

```diff
@@ -478,15 +479,11 @@ function MeasurementData (dataObject, Lt) {
           year_adjusted = this.points[i].year;
         };
 
-      } else if (this.points[i - 1].start) { // case 2: previous point is start
-        if (direction == forwardInTime) {
-          year_adjusted = this.points[i + 1].year;
-        } else if (direction == backwardInTime) {
+      } else if (this.points[i - 1].start || this.points[i].start) { // case 2: previous point is start
```

Comment:
the comment no longer seems to apply to this line. Is this actually fixing what you're trying to fix?

Reply:
The issue was which year the inserted point selected. My previous code was wrong and could be simplified to a single expression. The comment probably should be modified to 'previous or closest'.